docs/BR.md

```diff
@@ -1915,7 +1924,7 @@ If the Subordinate CA Certificate includes the id-kp-serverAuth extended key usa
 
 a. For each dNSName in permittedSubtrees, the CA MUST confirm that the Applicant has registered the dNSName or has been authorized by the domain registrant to act on the registrant's behalf in line with the verification practices of section 3.2.2.4.
 b. For each iPAddress range in permittedSubtrees, the CA MUST confirm that the Applicant has been assigned the iPAddress range or has been authorized by the assigner to act on the assignee's behalf.
-c. For each DirectoryName in permittedSubtrees the CA MUST confirm the Applicants and/or Subsidiary's Organizational name and location such that end entity certificates issued from the subordinate CA Certificate will be in compliancy with section 7.1.2.4 and 7.1.2.5.
+c. For each DirectoryName in permittedSubtrees the CA MUST confirm the Applicants and/or Subsidiary's Organizational name and location such that end entity certificates issued from the subordinate CA Certificate will be in compliance with section 7.1.2.4 and 7.1.2.5.
```

Comment:
```suggestion
c. For each DirectoryName in permittedSubtrees, the CA MUST confirm the Applicant's and/or Subsidiary's Organizational name and location such that end entity certificates issued from the subordinate CA Certificate will be in compliance with section 7.1.2.4 and 7.1.2.5.
```

Reply:
I think the intent here is to specify Affiliate, not necessarily "Subsidiary" specifically.

Reply:
I'm not sure? We should probably discuss more on list, but we don't really use Affiliate outside the context of either Affiliated with the CA or the case of Enterprise RAs and the O values. I can see why it might make sense, in the context of nameConstraints, to allow enumerating the Affiliates as an alternative to the Enterprise RA path, but have we seen that use case come up yet? I'd be inclined towards tackling it when we've got something concrete.